internal/provider/team_data_source_test.go

```diff
@@ -1,23 +1,61 @@
 package provider_test
 
-//func TestAccTeamDataSource(t *testing.T) {
-//	resource.Test(t, resource.TestCase{
-//		PreCheck:                 func() { testAccPreCheck(t) },
-//		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
-//		Steps: []resource.TestStep{
-//			// Read testing
-//			{
-//				Config: testAccTeamDataSourceConfig,
-//				Check: resource.ComposeAggregateTestCheckFunc(
-//					resource.TestCheckResourceAttr("data.cortex_team.platform_engineering", "tag", "platform_engineering"),
-//				),
-//			},
-//		},
-//	})
-//}
-//
-//const testAccTeamDataSourceConfig = `
-//data "cortex_team" "engineering" {
-//  tag = "platform_engineering"
-//}
-//`
+import (
+	"testing"
+
+	"github.com/cortexapps/terraform-provider-cortex/internal/cortex"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+type TestTeamDataSource struct {
+	Tag           string
+	Members       []cortex.TeamMember
+	SlackChannels []cortex.TeamSlackChannel
+}
+
+func (t *TestTeamDataSource) ToStubTeam() *cortex.Team {
+	return &cortex.Team{
+		TeamTag: t.Tag,
+		CortexTeam: cortex.TeamCortexManaged{
+			Members: t.Members,
+		},
+		SlackChannels: t.SlackChannels,
+	}
+}
+
+func testAccTeamDataSourceConfig(tag string) string {
+	return `
+data "cortex_team" "test-team" {
+  tag = "` + tag + `"
+}
+`
+}
+
+func TestAccTeamDataSource(t *testing.T) {
+	stub := TestTeamDataSource{
+		Tag: "test-team",
+		Members: []cortex.TeamMember{
+			{Name: "max", Email: "mcai@etsy.com"},
```

Comment:
maybe let's use a different email in the fixture

Reply:
replaced with `test@example.com`